sea-orm-codegen/src/entity/writer.rs

```diff
@@ -1182,7 +1213,7 @@ mod tests {
     #[test]
     fn test_gen_expanded_code_blocks() -> io::Result<()> {
         let entities = setup();
-        const ENTITY_FILES: [&str; 9] = [
```

Comment:
try [; _]

Reply:
I wish that works. Seems it's coming but not on stable Rust just yet
- https://github.com/rust-lang/rust/issues/85077